tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary.feature

```diff
@@ -244,31 +244,46 @@ Scenario: 9. Get the order summary that includes a list of additional services
         | additional-services | complete   | 2     |
         | funding-source      | incomplete |       |
 
+@5291
+Scenario:10. Get the order section status is set when conditions are met 
+    Given the user creates a new "<order-data-key>" order with id <order-id>
+    When the user makes a request to retrieve the order summary with the ID <order-id>
+    Then a response with status code 200 is returned
+    And the order Section Status is <section-status>
+    Examples: OrderData
+        | order-id   | order-data-key                                               | section-status |

```

Comment:
For consistency our headings are use PascalCase e.g. OrderId, OrderDataKey, SectionStatus